core/src/main/java/org/elasticsearch/index/query/QueryParser.java

```diff
@@ -27,14 +29,14 @@
 @FunctionalInterface
 public interface QueryParser<QB extends QueryBuilder> {
     /**
-     * Creates a new {@link QueryBuilder} from the query held by the {@link QueryParseContext}
-     * in {@link org.elasticsearch.common.xcontent.XContent} format
+     * Creates a new {@link QueryBuilder} from the query held by the
+     * {@link QueryParseContext} in
```

Comment:
This is not longer correct - more like "Creates a new QueryBuilder from the query represented by the xcontent in the parser." or something.